### PR TITLE
Fixed #3143: fixed functionality of 'show-weeks' attribute on datepicker popup

### LINKED ISSUE #3143: 'show-weeks' attribute not working on popup date picker
'show-weeks' attribute not working on popup date picker.

I tried to set 'show-weeks' to false and the calendar still show number of the week.

Your example with the flag show-weeks="false"

see http://plnkr.co/edit/oERqsVEJSRSQvrbuT1iH?p=preview

### PR DESCRIPTION
**Pull request by ankit5990**
Show weeks attribute was not working on datepicker popup.
Even when its value is set to 'false', the calender still shows weeks as per issue #3143.

The fixed version can be seen on plunkr
http://plnkr.co/edit/MJE44Blp7D2gIKWBOuwv?p=preview

### CODE REVIEW

**Comment by karianna** [comment]
@ankit5990  Merge conflict, can you rebase and push please.

**Comment by ankit5990** [comment]
ya sure lemme fix that. Its a pretty old PR so conflicts must have crept in...

**Comment by ankit5990** [comment]
Hi karianna, 
I have resolved the conflicts, please try now.

**Comment by chrisirhc** [comment]
Needs a test.

File: src/datepicker/datepicker.js [hunk 1]
```diff
@@ -529,6 +529,11 @@ function ($compile, $parse, $document, $position, dateFilter, dateParser, datepi
       if (attrs.dateDisabled) {
         datepickerEl.attr('date-disabled', 'dateDisabled({ date: date, mode: mode })');
       }
+
+      if(attrs.showWeeks) {
+        datepickerEl.attr('show-weeks',attrs.showWeeks);
```
Comment (chrisirhc) [inline src/datepicker/datepicker.js]: space after `,` and above, space before `(`;

**Comment by ankit5990** [comment]
@chrisirhc, yes I'll make the corrections and submit the code again with the required tests. Gimme a couple of days. I'm not very proficient with testing framework...

**Comment by ankit5990** [comment]
HI @chrisirhc, I have added the tests for the bug fix change. Please review. :)

**Comment by chrisirhc** [comment]
Squashed and merged. Also cleaned up trailing spaces. Thank you!

**Comment by ankit5990** [comment]
Thanks for merging the fix @chrisirhc :)

**Pull request closed without merging**